helm/values-dev.yaml

```diff
@@ -111,6 +111,9 @@ microservice-chart:
     TRANSACTION_REFUND_QUEUE_NAME: pagopa-d-weu-ecommerce-transactions-refund-queue
     TRANSACTION_NOTIFICATIONS_QUEUE_NAME: pagopa-d-weu-ecommerce-transaction-notifications-queue
     TRANSACTIONS_AUTHORIZATION_REQUESTED_QUEUE_NAME: pagopa-d-weu-ecommerce-transaction-auth-requested-queue
+    FEATURE_WALLET_USAGE_ENABLED: true
```

Comment:
add those new env variables to README to ([here](https://github.com/pagopa/pagopa-ecommerce-transactions-service/blob/main/README.md#environment-variables)) :)